packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -3,6 +3,7 @@ import { BaseComponent, DelayedRender, getId, classNamesFunction, getNativeProps
 import { IconButton } from '../../Button';
 import { Icon } from '../../Icon';
 import { IMessageBarProps, IMessageBarStyleProps, IMessageBarStyles, MessageBarType } from './MessageBar.types';
+import { getTheme } from '../../../../styling/lib';
```

Comment:
```suggestion
import { getTheme } from '../../Styling';
```

Reply:
Oops just saw this - fixed it with import { getTheme } from 'office-ui-fabric-react/lib/Styling';